app/code/Magento/InventoryCatalog/Test/Integration/AddStockStatusToSelectWithNotDefaultStockTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Test\Integration;
+
+use Magento\Catalog\Model\ResourceModel\Product\Collection;
+use Magento\CatalogInventory\Model\ResourceModel\Stock\Status as StockStatus;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Store\Model\Website;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test add stock status to select on not default website.
+ */
+class AddStockStatusToSelectWithNotDefaultStockTest extends TestCase
+{
+    /**
+     * @var StockStatus
+     */
+    private $stockStatus;
+
+    /**
+     * @var Website
+     */
+    private $website;
+
+    /**
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
+    /**
+     * @var null|string
+     */
+    private $storeCodeBefore = null;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->stockStatus = Bootstrap::getObjectManager()->create(StockStatus::class);
```

Comment:
Pls use get instead of create for Status object

Reply:
Done.